app/internal/place/client.go

```diff
@@ -33,13 +34,23 @@ type addressDetails struct {
 }
 
 type postcodeLookupResponse struct {
-	Results []ResultSet `json:"results"`
+	Results []ResultSet          `json:"results"`
+	Error   InvalidPostcodeError `json:"error"`
 }
 
 type ResultSet struct {
 	AddressDetails addressDetails `json:"DPA"`
 }
 
+type InvalidPostcodeError struct {
+	Statuscode int    `json:"statuscode,omitempty"`
+	Message    string `json:"message,omitempty"`
```

Comment:
`omitempty` is only relevant for things we serialise to JSON

Reply:
Agh, I always get this the wrong way round - thanks, will update.